src/test/java/uk/gov/hmcts/reform/sscs/personalisation/WithRepresentativePersonalisationTest.java

```diff
@@ -61,9 +69,10 @@ public void givenSscsCaseData_shouldSetRepresentativeNameIfPresent(
             .build();
         return new Object[]{
             new Object[]{sscsCaseDataWithReps, "Manish Sharma"},
+            new Object[]{sscsCaseDataWithRepsOrgOnly, "Sir / Madam"},
             new Object[]{sscsCaseDataWithNoReps, null},
             new Object[]{sscsCaseDataWithEmptyReps, null},
-            new Object[]{sscsCaseDataWithEmptyRepsAndEmptyNames, "null null"}
+            new Object[]{sscsCaseDataWithEmptyRepsAndEmptyNames, null}
```

Comment:
The original test of "null null" is what is returned if `representative.getName().getFirstName()` is null and `representative.getName().getLastName()` is null.  Are you sure you haven't weakened the test by setting the value to null?

Reply:
"null null" was a side-effect of the code performing an unintelligent string concat with nulls rather than "null null" being an AC that a dev had to write code for. Now that we're no longer flinging potential nulls at a string and seeing what sticks I think it's fair to remove it.